R/RcppExports.R

```diff
@@ -40,7 +40,13 @@ LL_SEVERE <- function() {
     .Call('_hector_LL_SEVERE', PACKAGE = 'hector')
 }
 
-#' @describeIn emissions Black carbon emissions (\code{"Tg"})
+#' @describeIn core Start of carbon tracking (Year)
```

Comment:
It should read 
```
#' @describeIn parameters Start of carbon tracking (Year)
```

there is no Roxygen documentation object called `core` so right now `help(TRACKING_DATE)` does nothing. It should work though after you make this change and rebuild the documentation.

Reply:
🎉 It does work now! Thank you :)